contact.md

```diff
@@ -1 +1,30 @@
-## Contact
\ No newline at end of file
+## Contact
+
+<table width="100%">
+	<tr>
+		<td>
+			<b>Project leads</b>
+		</td>
+		<td>
+			<a href="https://wiki.openmrs.org/display/~adamg">Adam Grzybkowski</a>, <a href="https://wiki.openmrs.org/display/~avijitghosh82">Avijit Ghosh</a>, <a href="https://wiki.openmrs.org/display/~tmarzeion">Tomasz Marzeion</a>
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<b>OpenMRS point of contact</b>
+		</td>
+		<td>
+			<a href="https://wiki.openmrs.org/display/~raff">Rafal Korytkowski</a> < rafal@openmrs.org >
```

Comment:
Maybe make the email address an `a` element as well with the `mailto` keyword:
 
    <a href="mailto:rafal@openmrs.org">rafal@openmrs.org</a>

Reply:
[Done](https://github.com/openmrs/openmrs-android-client-user-guide/pull/19/commits/a4bc50f3b07dffb8bfb79f1ebe7fd3acb704f2ad)